src/org/intellij/erlang/runconfig/ErlangRunConfigurationBase.java

```diff
@@ -125,9 +126,48 @@ public final RunningState getState(@NotNull Executor executor, @NotNull Executio
   protected abstract RunningState newRunningState(ExecutionEnvironment env, Module module);
 
   public static final class ErlangDebugOptions implements Serializable {
+    private boolean includeRebarDependencies = false;
+    private boolean fetchingDependencies = false;
     private boolean myAutoUpdateModulesNotToInterpret = true;
+
     private Set<String> myModulesNotToInterpret = new HashSet<String>();
 
+    private boolean loadingConfig = false;
+    private String appConfig = DEFAULT_CONFIG;
```

Comment:
These new options are not debug-specific - application and eunit run configuration should also employ them.